frontend/src/app/testQueue/TestCard/TestCard.test.tsx

```diff
@@ -0,0 +1,1442 @@
+            } as EDIT_QUEUE_ITEM_DATA,
+          },
+        },
+        {
+          request: {
+            query: SUBMIT_TEST_RESULT,
+            variables: {
+              patientId: testOrderInfo.patient.internalId,
+              deviceTypeId: device1Id,
+              specimenTypeId: specimen1Id,
+              results: [
+                { diseaseName: "COVID-19", testResult: "UNDETERMINED" },
+              ],
+              dateTested: null,
+            } as SUBMIT_QUEUE_ITEM_VARIABLES,
+          },
+          result: {
+            data: {
+              submitQueueItem: {
+                testResult: {
+                  internalId: testOrderInfo.internalId,
+                },
+                deliverySuccess: false,
+              },
+            } as SUBMIT_QUEUE_ITEM_DATA,
+          },
+        },
+      ];
+
+      const props = {
+        ...testProps,
+        testOrder: {
+          ...testProps.testOrder,
+          pregnancy: null,
+          symptomOnset: null,
+          noSymptoms: null,
+        },
+      };
+
+      const { user } = await renderQueueItem({ props, mocks });
+
+      // Select result
+      await user.click(
+        screen.getByLabelText("Inconclusive", {
+          exact: false,
+        })
+      );
+
+      // Submit
+      await user.click(screen.getByText("Submit results"));
+
+      await user.click(
+        screen.getByText("Submit anyway", {
+          exact: false,
+        })
+      );
+
+      // Displays submitting indicator
+      expect(
+        await screen.findByText(
+          "Submitting test data for Dixon, Althea Hedda Mclaughlin..."
+        )
+      ).toBeInTheDocument();
+
+      // Verify alert is displayed
+      await waitFor(() => {
+        expect(alertSpy).toHaveBeenCalledWith(
+          "The phone number provided may not be valid or may not be able to accept text messages",
+          "Unable to text result to Dixon, Althea Hedda Mclaughlin"
+        );
+      });
+      expect(
+        await screen.findByText(
+          "Unable to text result to Dixon, Althea Hedda Mclaughlin",
+          {
+            exact: false,
+          }
+        )
+      ).toBeInTheDocument();
+
+      // Submitting indicator and card are gone
+      expect(
+        await screen.findByText("Dixon, Althea Hedda Mclaughlin")
+      ).toBeInTheDocument();
+      expect(
+        await screen.findByText(
+          "Submitting test data for Dixon, Althea Hedda Mclaughlin..."
+        )
+      ).toBeInTheDocument();
+    });
+  });
+
+  it("updates custom test date/time", async () => {
+    const { user } = await renderQueueItem();
+    const toggle = await screen.findByLabelText("Current date and time");
+    await user.click(toggle);
+    const dateInput = screen.getByTestId("test-date");
+    expect(dateInput).toBeInTheDocument();
+    const timeInput = screen.getByTestId("test-time");
+    expect(timeInput).toBeInTheDocument();
+    user.type(dateInput, `${updatedDateString}T00:00`);
```

Comment:
these type methods probably need an `await`

Reply:
Turns out that `getByTestId` is synchronous so eslint warns against adding await here

Reply:
Updated to use await with `findByTestId` instead, thanks for spotting this!